src/main/java/org/opentripplanner/routing/request/AllowedModes.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.routing.request;
+
+import org.opentripplanner.model.TransitMode;
+
+import java.util.Set;
+
+public class AllowedModes {
```

Comment:
The `AllowedModes` is not a good name. The `Allowed` does not not tell what this is, maybe `RoutingRequestModes` is better?